render50

```diff
@@ -348,26 +348,27 @@ def concatenate(output, inputs):
 
     # Render blank page, inferring size from first input's first page
     temp = mkstemp()
-    size = "{}pt {}pt".format(readers[0].getPage(0).mediaBox[2], readers[0].getPage(0).mediaBox[3])
+    size = "{}pt {}pt".format(readers[0].pages[0].mediabox[2], readers[0].pages[0].mediabox[3])
     write(temp[1], [blank(size)], False)
-    page = PdfReader(temp[1]).getPage(0)
+    page = PdfReader(temp[1]).pages[0]
 
     # Concatenate files side by side
-    writer = PdfFileWriter()
+    writer = PdfWriter()
 
     # Concatenate pages
-    for i in range(max(map(lambda r: r.getNumPages(), readers))):
+    for i in range(max(map(lambda r: len(r.pages), readers))):
 
         # Leftmost page
-        left = copy(readers[0].getPage(i)) if i < readers[0].getNumPages() else copy(page)
+        left = copy(readers[0].pages[i]) if i < len(readers[0].pages) else copy(page)
 
         # Rightmost pages
         for reader in readers[1:]:
-            right = copy(reader.getPage(i)) if i < reader.getNumPages() else copy(page)
-            left.mergeTranslatedPage(right, left.mediaBox[2], 0, expand=True)
+            right = copy(reader.pages[i]) if i < len(reader.pages) else copy(page)
+            right.add_transformation(Transformation().translate(left.mediabox[2], 0))
```

Comment:
@dmalan Still trying to figure out why this new changes doesn't work. PDF side-by-side rendering only has the left half.

Reply:
Figured out, using the new method `merge_transformed_page()`